pages/content/migration-guidance.mdx

```diff
@@ -0,0 +1,504 @@
+---
+title: "Migrating from GOV.UK PaaS"
+---
+
+# Migrating from GOV.UK PaaS
+
+## Last updated
+
+- 1st November 2022 - first published
+
+## Government cloud hosting strategy
+
+The Government Digital Service (GDS) has provided GOV.UK PaaS since 2015, supporting and providing a public cloud platform for departments using a shared hosting and responsibility model. 
+
+Following an extensive analysis period, GDS has concluded that, while the platform has been successful in its aims, the underlying technology would now require investment before it could meet its goals in the long term. 
+
+Faced with this need for re-investment GDS has decided to [decommission the platform](https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/), in order to focus its budget and energy on other GDS products for common use by Government.
+
+GDS will not be providing a replacement hosting service.
+
+Tenants should look to their own department, or parent department, and use existing patterns and support available for public cloud hosting services. 
+
+In deciding where to host their service next, tenants should consider the following principles from the [Central Digital & Data Office (CDDO)](https://www.gov.uk/government/organisations/central-digital-and-data-office/about#responsibilities):
+
+- Tenants should continue to use public cloud services, or internal platforms based on public cloud offerings. These continue to be the platforms of choice for government services.
+- The [Central Digital & Data Office (CDDO)](https://cddo.blog.gov.uk/) has secured Director General level agreement to help share services. Larger departments are prepared to share knowledge, patterns and platforms with the rest of government. If tenants have no existing cloud engineering capability, you could approach the CDDO, via the GOV.UK PaaS team [support form](https://admin.london.cloud.service.gov.uk/support/contact-us), to benefit from the sharing agreements.
```

Comment:
The PaaS team are not going to be a mail forwarding service for an unknown service offering to an unknown entity.

Reply:
there was a lot of debate about how to handle this - Phil's team suggested an individual's email address we can use while they set up a shared inbox.

Reply:
ok have replaced the link to the team form with a mailto: link